projects/onto-search/src/lib/onto-search-facet/range-facet/range-slider/range-slider.component.ts

```diff
@@ -51,6 +56,25 @@ export class RangeSliderComponent implements OnInit, AfterViewInit, OnDestroy, O
   public maxElement;
 
   ngOnInit(): void {
+    this.initializeData();
+  }
+
+  public ngAfterViewInit(): void {
+    this.minElement = this.minSlider.nativeElement;
+    this.maxElement = this.maxSlider.nativeElement;
+
+    this.init(this.selectedMin, this.selectedMax);
+    this.onResize = (): void => this.update(this.parseInt(this.maxElement.getAttribute('max')));
+  }
+
+  public ngOnChanges(changes: SimpleChanges): void {
```

Comment:
Same as [here](https://github.com/Ontotext-AD/ontotext-reusable-ui-components/pull/39/files#r538093687)

Reply:
Same as the other one.